chore: add introduction

dekatron opened the pull request:
Add a introduction file to the repo for code school "Mastering GitHub" course. The course is Japanese themed so lets write the message in Japanese.

dekatron committed:
chore: add introduction

Add a introduction file to the repo for code school "Mastering GitHub" course. The course is Japanese themed so lets write the message in Japanese.

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,2 @@
+はじめまして、ユージーンともうします。
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

dekatron committed:
chore: Add a code school path name

can be any of Ruby, JavaScript, HTML & CSS, iOS or Electives... just so long as it's in English.
